src/plugins/plot/chart/MctChart.vue

```diff
@@ -418,7 +421,10 @@ export default {
             // Have to throw away the old canvas elements and replace with new
             // canvas elements in order to get new drawing contexts.
             const div = document.createElement('div');
-            div.innerHTML = this.canvasTemplate + this.canvasTemplate;
+            div.innerHTML = `
+            <canvas style="position: absolute; background: none; width: 100%; height: 100%;"></canvas>
```

Comment:
what about this innerhtml?

Reply:
The vulnerability arises not just from assignment to innerHTML, but when that assignment contains variables that could potentially store user input. It can expose the app at some level to cross site scripting. So this one should be OK.

Reply:
Got it